### PR TITLE
Monitior Section storage and use joins_allowed switch

### PR DESCRIPTION
**Pull request by Yoga07**

### CODE REVIEW

File: src/node/mod.rs [hunk 1]
```diff
@@ -92,14 +92,32 @@ impl Node {
...
+        } else {
+            info!("We are Infant");
+            Infant
+        };
+
+        let mut duties = NodeDuties::new(node_info, network_api.clone()).await;
         let _ = match age_group {
-            Infant => Ok(NodeOperation::NoOp),
+            Infant => {
+                info!("Do nothing as Infant");
```
Comment (joshuef) [inline src/node/mod.rs:111]:
```suggestion
                info!("Becoming Infant, do nothing.");
```

File: src/node/node_duties/messaging/network_sender.rs [hunk 2]
```diff
@@ -111,6 +111,7 @@ impl NetworkSender {
             Address::Node(xorname) => DstLocation::Node(xorname),
             Address::Client(xorname) | Address::Section(xorname) => DstLocation::Section(xorname),
         };
+        info!("Dst Location: {:?}", dst);
```
Comment (joshuef) [inline src/node/node_duties/messaging/network_sender.rs:114]:
```suggestion
        info!("Destination: {:?}", dst);
```

File: src/node/node_duties/mod.rs [hunk 3]
```diff
@@ -108,9 +108,29 @@ impl NodeDuties {
                     .await
             }
             NoOp => Ok(NodeOperation::NoOp),
+            StorageFull => self.notify_section_of_our_storage().await,
         }
     }
 
+    async fn notify_section_of_our_storage(&mut self) -> Result<NodeOperation> {
```
Comment (joshuef) [inline src/node/node_duties/mod.rs:115]: If this is never used but to notify full storage, then I'd rename this to reflect that

**Review by joshuef — CHANGES_REQUESTED** [review]
minor tweaks on some log messages and naming

File: src/node/elder_duties/key_section/payment/mod.rs [hunk 4]
```diff
@@ -0,0 +1,148 @@
+// Copyright 2020 MaidSafe.net limited.
```
Comment (oetyng) [inline src/node/elder_duties/key_section/payment/mod.rs:1]: This file can be removed, was merged with transfers.

File: src/node/elder_duties/key_section/transfers/replica_manager.rs [hunk 5]
```diff
@@ -398,4 +398,4 @@
 //     pub fn debit_without_proof(&mut self, transfer: Transfer) -> NdResult<()> {
```
Comment (oetyng) [inline src/node/elder_duties/key_section/transfers/replica_manager.rs:398]: This file can be removed as well.

File: src/node/node_duties/mod.rs (hunk 3 above)
Comment (oetyng) [reply src/node/node_duties/mod.rs:115]: I'd say, additionally, that this is an event, not a cmd. `StorageIsFull` is a notification about a fact. It's not a cmd to do something.
But that can be fixed in a follow up PR.

**Pull request merged**